main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Commands/ProjectCommands.cs

```diff
@@ -89,7 +89,8 @@ public enum ProjectCommands
 		SelectActiveRuntime,
 		EditSolutionItem,
 		Unload,
-		SetStartupProjects
+		SetStartupProjects,
+		AddEmptyClass
```

Comment:
Style nit:
```suggestion
		AddEmptyClass,
```

Even though the IDE might show it as redundant, it makes it easier for future diffs if we just add a trailing comma, as the next change will just add one line, instead of adding one and modifying one.